test_suite/generic/test_generic.py

```diff
@@ -586,7 +586,7 @@ def test_eth0_network_adapter_setup(self, host):
                 f'Unexpected device name. Expected: "{device_name}"'
 
     @pytest.mark.run_on(['rhel'])
-    @pytest.mark.exclude_on(['rhel7.9', 'rhel8.4'])
+    @pytest.mark.exclude_on(['<rhel8.5'])
```

Comment:
seems like the test "test_network_manager_cloud_setup" passed on rhel-8.4 -  but it should be skipped no?

Reply:
I noticed to an issue with the regex of the  exclude_on marker will create a PR to fix it - this is why it didn't skip

Reply:
https://github.com/osbuild/cloud-image-val/pull/195